src/Codeception/Module/AMQP.php

```diff
@@ -90,6 +91,18 @@ public function _initialize()
         }
     }
 
+    /**
+     * Add a queue to purge list
+     *
+     * @param string $queue
+     */
+    public function addQueue($queue)
```

Comment:
`addQueue` sounds too general and doesn't represent the effect taken. 
Maybe it should be `scheduleQueueCleanup` or something like that

Reply:
ok :) I didn't have good idea for the name.

Reply:
done !